src/en/styles/spacing/spacing.md

```diff
@@ -9,7 +9,7 @@ eleventyNavigation:
   order: 2
   description: Spacing tokens help position elements onscreen and in components.
   thumbnail: /images/en/foundations/preview/preview-spacing.svg
-  alt:
+  alt: A red line vertical line with two darker horizontal lines at either end shows the size of the token, labelled to its right --gcds-spacing-500.
```

Comment:
I compared this with the spreadsheet and it's correct (exactly the same), but reading it -- is this a mistake or intentional?
```suggestion
  alt: A red vertical line with two darker horizontal lines at either end shows the size of the token, labelled to its right --gcds-spacing-500.
```

Reply:
Nope that is a mistake